<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="param_path" default="$(find-pkg-share autoware_control_center)/config/control_center.param.yaml"/>

  <node pkg="autoware_control_center" exec="autoware_control_center_node" name="control_center" output="screen">
    <param from="$(var param_path)"/>
  </node>
</launch>
